holoviews/element/path.py

```diff
@@ -193,22 +205,36 @@ class Ellipse(BaseShape):
 
     height = param.Number(default=1, doc="The height of the ellipse.")
 
-    aspect= param.Number(default=1.0, doc="The aspect ratio of the ellipse.")
+    width = param.Number(default=1, doc="The width of the ellipse.")
+
+    aspect= param.Number(default=1.0, doc="""
+       Final divisor only applied to the final height.""")
 
```

Comment:
I probably wouldn't recommend using ``aspect`` anymore tbh and would consider deprecating it. I am only leaving it in right now for backwards compatibility. Should I issue a deprecation warning when it is used?